grpc/cpp-cubic/README.md

```diff
@@ -0,0 +1,78 @@
+# Cubic C++ SDK
+Unlike other languages, C++ has more strict requirements to
+integrate with gRPC. The easiest way to accomodate these
+requirements is to build gRPC as part of the C++ project
+and use it to generate the required C++ files from the 
+cubic.proto file.
+
+## CMake Build
+To help simplify the build process, this project uses
+[CMake](www.cmake.org). CMake will automatically download
+the grpc source code and include it as a subproject, giving
+access to the grpc libraries and the protoc compiler.
+
+To build as a standalone library execute the following
+commands:
+```bash
+# Create a build directory. It can be named anything and
+# can exist outside of the source code directory.
+mkdir build-cubic-client && cd build-cubic-client
+
+# Run CMake to download grpc and generate makefiles.
+# The final path specifies the directory that contains the
+# CMakeLists.txt file for the cubic_client C++ project.
+cmake -DCMAKE_BUILD_TYPE=Release <path/to/sdk-cubic/grpc/cpp-cubic>
+make cubic_client
+
+# To make a shared library, add -DBUILD_SHARED_LIBS=TRUE when running cmake
```

Comment:
We might want to clarify that this is a separate option, not that they'd run the steps twice.  Maybe something like
```
# Run CMake to download grpc and generate makefiles.
# To make a shared library, add -DBUILD_SHARED_LIBS=TRUE
# The final argument specifies the directory that contains the
# CMakeLists.txt file for the cubic_client C++ project.
cmake -DCMAKE_BUILD_TYPE=Release <path/to/sdk-cubic/grpc/cpp-cubic>
# OR
cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=TRUE <path/to/sdk-cubic/grpc/cpp-cubic>

make cubic_client
```